teuthology/nuke.py

```diff
@@ -400,19 +400,24 @@ def check_console(hostname):
         host=shortname,
         domain=config['ipmi_domain'])
     log.info('checking console status of %s' % cname)
-    if not console.check_status():
-        # not powered on or can't get IPMI status.  Try to power on
-        console.power_on()
```

Comment:
ew.  I didn't know nuke did this.  That's unintuitive to me, and you're not changing it, but ... blech.